scripts/2.mutation-classifier.py

```diff
@@ -133,11 +124,10 @@
 
 # In[10]:
 
-
 # Typically, this type of split can only be done 
 # for genes where the number of mutations is large enough
 X = pd.concat([covariate_df, expression_df], axis='columns')
-X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=0)
+X_train, X_test, y_train, y_test = train_test_split(X, y, stratify=y, test_size=0.1, random_state=0)
```

Comment:
Great! It'd also be nice to stratify by disease type, but only if there is an elegant implementation. Can also do this a later PR.

Reply:
I'll look into this in a future PR. Based on a quick review it looks like multi-column stratify isn't supported in scikit-learn 18.x (cognoma is currently using 18.1) but was added/fixed in 19.0:
https://github.com/scikit-learn/scikit-learn/pull/9044
https://github.com/scikit-learn/scikit-learn/issues/9037

Reply:
Nice! In a future upgrade, we may want to consider upgrading everything as much as possible... and could also make this change.